### DISCUSSION TITLE
2006/06/16/%E5%88%B6%E4%BD%9C%E8%87%AA%E5%B7%B1%E7%9A%84%E5%8D%A1%E9%80%9A%E5%A4%B4%E5%83%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#85] [Announcements]
# 2006/06/16/%E5%88%B6%E4%BD%9C%E8%87%AA%E5%B7%B1%E7%9A%84%E5%8D%A1%E9%80%9A%E5%A4%B4%E5%83%8F

MSN现在提供了一个可以制作卡通头像的网站：

https://blog.qizhen.xyz/2006/06/16/%E5%88%B6%E4%BD%9C%E8%87%AA%E5%B7%B1%E7%9A%84%E5%8D%A1%E9%80%9A%E5%A4%B4%E5%83%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Y
2006年07月7日 12:45 编辑
挺有意思的~

回复
Qizhen
2006年06月19日 22:24 编辑
我也觉得不太像

回复
yoyo
2006年06月19日 22:16 编辑
看着不怎么像阿

回复
Amy
2006年06月19日 10:29 编辑
thanks for coming…. nice to meet you and hello from HK…

回复
ka
2006年06月17日 22:25 编辑
拜访一下，谢谢你的留言

回复
Yoyo
2006年06月17日 21:56 编辑
眼镜大哥！

回复
xiaoji
2006年06月17日 19:43 编辑
嘻嘻，你叫他QQ哥就行啦^^

回复
璇璇
2006年06月17日 15:07 编辑
这里很特别也很简洁，就像你给我的留言一样呵呵，不错哦加油，嘻嘻你是哥哥还是姐姐？

回复
小蝶▽
2006年06月17日 08:27 编辑
我也来转转。。呵呵。。

回复
